### PR TITLE
Remove the ansible warnings from the installation

### LINKED ISSUE #20: Remove the ansible warnings from the installation
RHEL 7 has a newer version of ansible than centos 6/7/ and rhel 6.  It recently moved to ansible version 2.8.0.  When this happened, we started to see a lot of deprecation warnings like the following:
```
[DEPRECATION WARNING]: evaluating is_rhel_centos_6 as a bare variable, this behaviour will go away and you might need to add 
|bool to the expression in the future. Also see CONDITIONAL_BARE_VARS configuration toggle.. This feature will be removed in 
version 2.12. Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
skipping: [localhost] => {"changed": false, "skip_reason": "Conditional result was False"}
```

This purpose of this issue is to fix ALL warnings that we can (not just the deprecation warning).  Attached is a RHEL 7 log file for reference.

### PR DESCRIPTION
**Pull request by Dostonbek1**
Removed the ansible-related warnings

closes #20

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[[source]]
+name = "pypi"
```
Comment (myersCody) [inline Pipfile]: We don't need a pipfile for the `quipucords-installer` because we don't test locally, we test through the vagrant machines.

**Comment by myersCody** [comment]
I found two additional warnings while testing: 
command: `make install-offline version=0.0.46`
```
TASK [epel : Check for EPEL rpm (RHEL/Centos)] ***********************************************************************************************************************************************************************************************
 [WARNING]: Consider using yum, dnf or zypper module rather than running rpm

fatal: [localhost]: FAILED! => {"changed": true, "cmd": "rpm -qa | grep epel-release", "delta": "0:00:00.457317", "end": "2019-06-11 14:56:36.589256", "msg": "non-zero return code", "rc": 1, "start": "2019-06-11 14:56:36.131939", "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
...ignoring
```
command: `make install-version version=0.0.46 [on rhel7]`
```
TASK [qpc_server : Open port in firewall on host machine for Quipucords server] **************************************************************************************************************************************************************
 [WARNING]: The value 9443 (type int) in a string field was converted to u'9443' (type string). If this does not look like what you expect, quote the entire value to ensure it does not change.

changed: [localhost] => {"chain": "INPUT", "changed": true, "flush": false, "ip_version": "ipv4", "rule": "-p tcp -j ACCEPT --destination-port 9443", "state": "present", "table": "filter"}
```

**Comment by Dostonbek1** [comment]
> I found two additional warnings while testing:
> command: `make install-offline version=0.0.46`
> 
> ```
> TASK [epel : Check for EPEL rpm (RHEL/Centos)] ***********************************************************************************************************************************************************************************************
>  [WARNING]: Consider using yum, dnf or zypper module rather than running rpm
> 
> fatal: [localhost]: FAILED! => {"changed": true, "cmd": "rpm -qa | grep epel-release", "delta": "0:00:00.457317", "end": "2019-06-11 14:56:36.589256", "msg": "non-zero return code", "rc": 1, "start": "2019-06-11 14:56:36.131939", "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
> ...ignoring
> ```
> 
> command: `make install-version version=0.0.46` [on rhel7]
> 
> ```
> TASK [qpc_server : Open port in firewall on host machine for Quipucords server] **************************************************************************************************************************************************************
>  [WARNING]: The value 9443 (type int) in a string field was converted to u'9443' (type string). If this does not look like what you expect, quote the entire value to ensure it does not change.
> 
> changed: [localhost] => {"chain": "INPUT", "changed": true, "flush": false, "ip_version": "ipv4", "rule": "-p tcp -j ACCEPT --destination-port 9443", "state": "present", "table": "filter"}
> ```



> I found two additional warnings while testing:
> command: `make install-offline version=0.0.46`
> 
> ```
> TASK [epel : Check for EPEL rpm (RHEL/Centos)] ***********************************************************************************************************************************************************************************************
>  [WARNING]: Consider using yum, dnf or zypper module rather than running rpm
> 
> fatal: [localhost]: FAILED! => {"changed": true, "cmd": "rpm -qa | grep epel-release", "delta": "0:00:00.457317", "end": "2019-06-11 14:56:36.589256", "msg": "non-zero return code", "rc": 1, "start": "2019-06-11 14:56:36.131939", "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
> ...ignoring
> ```
> 
> command: `make install-version version=0.0.46` [on rhel7]
> 
> ```
> TASK [qpc_server : Open port in firewall on host machine for Quipucords server] **************************************************************************************************************************************************************
>  [WARNING]: The value 9443 (type int) in a string field was converted to u'9443' (type string). If this does not look like what you expect, quote the entire value to ensure it does not change.
> 
> changed: [localhost] => {"chain": "INPUT", "changed": true, "flush": false, "ip_version": "ipv4", "rule": "-p tcp -j ACCEPT --destination-port 9443", "state": "present", "table": "filter"}
> ```

I was able to take out the first warning, but I'm not sure how to go about the second one.

File: install/roles/qpc_server/tasks/main.yml [hunk 2]
```diff
@@ -296,32 +298,38 @@
...
   fail:
     msg: "The Postgres DB's docker image was not found in the output of the docker images command."
   when:
-    - load_docker_postgres_attempted
+    - load_docker_postgres_attempted|bool
     - not postgres_docker_image_present
 
 - name: Make server home directory
   shell: mkdir -p "{{server_install_dir}}"
+  args:
```
Comment (myersCody) [inline install/roles/qpc_server/tasks/main.yml]: I don't think we should silence these warning by turning them off. 

For example, Ansible does have a better way of [creating directories](https://docs.ansible.com/ansible/latest/modules/file_module.html) than passing `mkdir` through a shell command. 

I think you should see if you can find a way to improve or rewrite the code so that we do not have to use `args: warn: false` at all, or only as a last resort.

File: install/roles/qpc_server/tasks/main.yml [hunk 3]
```diff
@@ -296,32 +298,38 @@
...
+    warn: false
 
 - name: Make server log directory
   shell: mkdir -p "{{server_install_dir}}/log"
+  args:
```
Comment (myersCody) [inline install/roles/qpc_server/tasks/main.yml]: Same as the comment above, but for all `args: warn: false`.

File: install/roles/qpc_server/tasks/main.yml (hunk 2 above)
Comment (kholdaway) [reply install/roles/qpc_server/tasks/main.yml]: Agree with @myersCody

File: install/roles/qpc_server/tasks/main.yml (hunk 3 above)
Comment (kholdaway) [reply install/roles/qpc_server/tasks/main.yml]: Agree with @myersCody

File: install/roles/epel/tasks/main.yml [hunk 4]
```diff
@@ -58,23 +60,29 @@
...
+    state: present
   become: true
   when:
     - not have_epel_rpm
-    - find_epel_rpm_local
+    - find_epel_rpm_local|bool
 
 - name: Install EPEL for RHEL 6 from internet (RHEL/Centos 6)
-  shell: rpm -Uvh  --force https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm
+  yum:
```
Comment (myersCody) [inline install/roles/epel/tasks/main.yml]: This should have conditions so that it only runs on rhel6 or centos6

File: install/roles/epel/tasks/main.yml [hunk 5]
```diff
@@ -58,23 +60,29 @@
...
+    name: https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm
+    state: present
   become: true
   when:
     - not have_epel_rpm
 
 - name: Install EPEL for RHEL 7 from internet (RHEL/Centos 7)
-  shell: rpm -Uvh --force https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+  yum:
```
Comment (myersCody) [inline install/roles/epel/tasks/main.yml]: This should have a condition so that it only runs on rhel7 or centos7

File: install/roles/epel/tasks/main.yml [hunk 6]
```diff
@@ -58,23 +60,35 @@
...
+    name: https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+    state: present
   become: true
+  ignore_errors: yes
   when:
+    - is_centos_7|bool
```
Comment (abaiken) [inline install/roles/epel/tasks/main.yml]: I think this should be the `is_rhel_centos_7` instead of `is_centos_7`

File: install/roles/qpc_server/tasks/main.yml [hunk 7]
```diff
@@ -353,13 +361,13 @@
   shell: docker run --name qpc-db  -e POSTGRES_USER="{{dbms_user}}" -e POSTGRES_PASSWORD="{{dbms_password}}" -v /var/lib/docker/volumes/qpc-data:/var/lib/postgresql/data -d postgres:"{{POSTGRES_VERSION}}"
   become: true
   when:
-    - is_rhel_centos_6
+    - is_rhel_centos_6|bool
 
 - name: Start Postgres container with docker volume (RHEL 7, Centos 7)
-  shell: docker run --name qpc-db  -e POSTGRES_USER="{{dbms_user}}" -e POSTGRES_PASSWORD="{{dbms_password}}" -v qpc-data:/var/lib/postgresql/data -d postgres:"{{POSTGRES_VERSION}}"
+  shell: docker run --name qpc-db  -e POSTGRES_USER="{{dbms_user}}" -e POSTGRES_PASSWORD="{{dbms_password}}" -v /var/lib/docker/volumes/qpc-data:/var/lib/postgresql/data -d postgres:"{{POSTGRES_VERSION}}"
```
Comment (abaiken) [inline install/roles/qpc_server/tasks/main.yml]: Why did you change this?

File: install/roles/epel/tasks/main.yml [hunk 8]
```diff
@@ -16,15 +16,14 @@
...
 - name: Set have_rpm
   set_fact:
-    have_rpm: "{{ have_rpm_raw.rc == 0 }}"
+    have_rpm: "{{ have_rpm_raw.rc == 0 }}" 
 
 - name: Check for EPEL rpm (RHEL/Centos)
   shell: rpm -qa | grep epel-release
   register: have_epel_rpm_raw
   become: true
-  ignore_errors: yes
```
Comment (abaiken) [inline install/roles/epel/tasks/main.yml]: I think removing this causes it to fail on centos7

File: install/roles/epel/tasks/main.yml [hunk 9]
```diff
@@ -16,15 +16,14 @@
 
 - name: Set have_rpm
   set_fact:
-    have_rpm: "{{ have_rpm_raw.rc == 0 }}"
+    have_rpm: "{{ have_rpm_raw.rc == 0 }}"
```
Comment (myersCody) [inline install/roles/epel/tasks/main.yml]: You should remove the whitespace from the end of this sentence.
